solr/core/src/test/org/apache/solr/update/processor/NestedAtomicUpdateTest.java

```diff
@@ -709,6 +709,67 @@ private void testBlockAtomicSetToNullOrEmpty(boolean empty) throws Exception {
         "/response/docs/[0]/cat_ss/[1]==\"ccc\"");
   }
 
+  @Test
+  public void testBlockAtomicUpdateWithoutTouchingLatLonPointSpartialField() throws Exception {
```

Comment:
This test appears to be a copy-paste from the one above, but simply adding a latLon field to it.  Nothing more.  Right?  Then lets not approach it this way; we only need one method here.  Comments inside can mention that latLon is here due to this issue (reference JIRA issue).  If you can make this change quickly then it can get into 8.4.